### DISCUSSION TITLE
What determines what emoji are available in kitty's unicode picker?

### DISCUSSION THREAD

**Discussion by lokesh-krishna** [kovidgoyal/kitty#3979] [Q&A] [answered]
I have `ttf-twemoji` installed and all supported emoji show up as expected when pasted in kitty but kitty's unicode picker's emoji section only offers me this limited selection:
![image](https://user-images.githubusercontent.com/20235646/131249000-b83bf4b9-a48a-4892-b783-29c198b88f38.png)

What can I do to have the full gamut of Unicode emoji show up here?

**Comment by kovidgoyal** [comment, marked as answer]
On Sun, Aug 29, 2021 at 04:36:00AM -0700, Lokesh Krishna wrote:
> I have `ttf-twemoji` installed and all supported emoji show up as expected when pasted in kitty but kitty's unicode picker's emoji section only offers me this limited selection:
> ![image](https://user-images.githubusercontent.com/20235646/131249000-b83bf4b9-a48a-4892-b783-29c198b88f38.png)

That is just a list of common emoji. Use the name tab and type in your
emoji name to pick it.

**Reply by lokesh-krishna** [reply]
Oh, alright :)
Not related but is is possible to copy the selected emoji automatically to the clipboard?

**Reply by kovidgoyal** [reply]
No, not really, the unicode input kitten is meant for inputting into kitty, not other programs. It should be easy enough to add an option to it to copy the selected char to the clipboard instead, a PR is welcome.

**Comment by lokesh-krishna** [comment]
What browser do you use to be able to mark an answer as a solution? Can't from qutebrowser, Brave or Firefox :/

**Reply by kovidgoyal** [reply]
https://github.com/kovidgoyal/vise
